Dockerfile

```diff
@@ -0,0 +1,57 @@
+FROM jupyter/base-notebook:eb70bcf1a292
```

Comment:
We can use this other jupyter-neuron docker now as a starting point (what you asked at the meeting that is now available) https://github.com/MetaCell/NetPyNE-UI/blob/refactor-dockerimage/Dockerfile

Reply:
Done + I added a comment to make it obvious to everyone, that this Dockerfile is just a template and is not intended to be functional at the moment.